decomposed/extensions/MDDE_LDM/Profile/Entity/Stereotypes/mdde_BusinessRule/Stereotypes/mdde_BusinessRuleWithMapping/Stereotypes/mdde_AggregateBusinessRule/Event_Handlers/CanCreate.xml

```diff
@@ -0,0 +1,25 @@
+<o:EventHandlerTargetItem Id="7768AB67-15D8-4C68-8754-5697D2960D23">
+<a:ObjectID>7768AB67-15D8-4C68-8754-5697D2960D23</a:ObjectID>
+<a:Name>CanCreate</a:Name>
+<a:CreationDate>1726842611</a:CreationDate>
+<a:Creator>RichaneRamsoedit</a:Creator>
+<a:TargetCategory.Type>1</a:TargetCategory.Type>
+<a:HandlerScript>Function %CanCreate%(parent)
+   Dim model : Set model = parent.Model
+   
+   &#39;Foutafhandeling wanneer er geen model is
+   If model Is Nothing Then

```

Comment:
Did this scenario occur in your PowerDesigner session? Cause I can't imagine there being an event without a Model.